reference/dtr/2.6/cli/backup.md

```diff
@@ -27,7 +27,8 @@ docker run -i --rm --log-driver none docker/dtr:{{ page.dtr_version }} \
 {% raw %}
 ```bash
 DTR_VERSION=$(docker container inspect $(docker container ps -f \
```

Comment:
In the case of using this with OSX I think it may be helpful to specify that your client bundle should be sourced first. 

As a side note I tried to run this on my mac with a client bundle, but it didn't work.

Here is the output for reference: 
```
DTR_VERSION=$(docker container inspect $(docker container ps -f \
  name=dtr-registry -q) | grep -m1 DTR_VERSION | \
  sed 's/.*DTR_VERSION=\([0-9].[0-9].[0-9]\)",/\1/'); \
REPLICA_ID=$(docker ps --filter name=dtr-rethinkdb \
  --format "{{ .Names }}" | head -1 | sed 's|.*/||' | sed 's/dtr-rethinkdb-//'); \
read -p 'ucp-url (The UCP URL including domain and port): ' UCP_URL; \
read -p 'ucp-username (The UCP administrator username): ' UCP_ADMIN; \
read -sp 'ucp password: ' UCP_PASSWORD; \
docker run --log-driver none -i --rm \
  --env UCP_PASSWORD=$UCP_PASSWORD \
  docker/dtr:$DTR_VERSION backup \
  --ucp-username $UCP_ADMIN \
  --ucp-url $UCP_URL \
  --ucp-ca "$(curl https://${UCP_URL}/ca)" \
  --existing-replica-id $REPLICA_ID > \
  dtr-metadata-${DTR_VERSION}-backup-$(date +%Y%m%d-%H_%M_%S).tar
read: -p: no coprocess
read: -p: no coprocess
read: -p: no coprocess
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: ca
docker: invalid reference format.
See 'docker run --help'.
```